benchmarks/arrow_parquet-arrow-fuzz/benchmark.yaml

```diff
@@ -1,5 +1,7 @@
+commit: e9222ae00b515a0f57d8c7d54bdb19457d68582f
```

Comment:
are these the old commits?

Reply:
>They are old enough to have multiple known bugs to be discovered by fuzzers.

Ack. thanks
Are these bugs from a single fuzz target? Otherwise they aren't useful.

Reply:
Ah, that is a good point, I will double-check when adding new bug-based benchmarks.

For these 5 benchmarks:
1. `arrow`: one UBSAN error from `parquet-arrow-fuzz`.
2. `aspell`: one UBSAN error from `aspell_fuzzer `.
3. `ffmpeg`: one ASAN error from `ffmpeg_DEMUXER_fuzzer`.
4. `file`: 20+ ASAN&UBSAN errors from `magic_fuzzer`.
5. `grok`: 14 ASAN&UBSAN errors from `grk_decompress_fuzzer`.

I wonder if the first 2 are useful as I reckon we don't have `UBSAN` in FuzzBench, right?

Reply:
We do have UBSAN but one bug is probably not enough

Reply:
I think the `file` fuzzers do crazy things with processes not sure if it's good to use them.

Reply:
I see, I will add some alternatives, then : )

Reply:
It might be fine. I really am not sure.

Reply:
No problem, I think we do plan to gradually revive more in 1-2 months so I am more than happy to do that earlier.
BTW, would you mind me asking how we are using UBSAN?
I thought [it requires `-fsanitize=undefined`](https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html#usage), but I did not find that flag in our code...

Reply:
we pick the individual parts of UBSAN ourselves. e.g. integer-overflow

Reply:
i would make sure to do a bunch of experiments with a benchmark before including it in a public competition

Reply:
Good point, I will start one to double-check the current 5, then : )

Reply:
Launched in #1589.